financialaid/constants.py

```diff
@@ -222,3 +222,19 @@
     "UG": 0,
     "ZW": 0
 }
+
+FINANCIAL_AID_APPROVAL_MESSAGE_BODY = """Your application for financial aid has been approved!
+Please log on to your account for more details.
+Love,
+MIT Micromasters"""
+FINANCIAL_AID_REJECTION_MESSAGE_BODY = """Your application for financial aid has been rejected.
+Please log on to your account for more details.
+Sympathy,
+MIT Micromasters"""
+FINANCIAL_AID_DOCUMENTS_MESSAGE_BODY = """Your financial aid documents have been received.
```

Comment:
That makes sense to me.  These are placeholders since I don't think we've discussed the content of the emails yet.